desci-server/src/controllers/data/update.ts

```diff
@@ -24,7 +24,10 @@ export interface ErrorResponse {
 export const update = async (req: RequestWithNode, res: Response<UpdateResponse | ErrorResponse | string>) => {
   const owner = req.user;
   let node = req.node;
-  const { uuid, manifest: draftManifest, contextPath, componentType, componentSubtype, newFolderName } = req.body;
+  const { uuid, manifest: draftManifest, componentType, componentSubtype, newFolderName } = req.body;
+  let { contextPath } = req.body;
```

Comment:
nitpick but I'm not a fan of overwriting variables, if I saw this somewhere and I see it's passed in the request body I'd assume it hasn't been messed with :eyes: 

I'd deconstruct like the others, then bind the result of the clean to `const cleanContextPath` or smt :angel: